recipe/bld.bat

```diff
@@ -0,0 +1,2 @@
+SET CMAKE_ARGS="%CMAKE_ARGS% -DUSE_BUNDLED_BLAS_WIN:BOOL=FALSE"
```

Comment:
```suggestion
SET "CMAKE_ARGS=%CMAKE_ARGS% -DUSE_BUNDLED_BLAS_WIN:BOOL=FALSE"
```

CMD quoting rules are tricky. This is a bit more robust, but you might see issues with nested quotes. Keep an eye.

Reply:
I had nested quotes with the version you commented on, which did not work, but then it worked with no quotes. I will try your suggestion.

Reply:
By the way, is this how you set `CMAKE_ARGS` on Windows when not invoking `cmake` directly? scikit-build's documentation suggests using `python setup.py ... <cmake args>` but I want to keep `pip install` as more future-proof (not sure if `python setup.py` still works any way). The alternative way to set CMake options is to use `CMAKE_ARGS` but conda-forge's build set up overrides `CMAKE_ARGS` when I try to set it in the recipe, so I moved it down in the script like this, so I could extend what conda-forge set after it set it.

Reply:
Yes, conda-forge infra sets `CMAKE_ARGS` for you. It looks like the current approach works so... yay? 🥳

Reply:
Yes, it's good now :slightly_smiling_face: I just wasn't sure if dropping to bld.bat and extending `CMAKE_ARGS` was idiomatic for conda-forge or if there were some other way to extend `CMAKE_ARGS`.